.github/workflows/codeql_analysis.yml

```diff
@@ -29,6 +29,11 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v3
 
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: 1.21.5
+
```

Comment:
Moved this, see the warning at https://github.com/vitessio/vitess/security/code-scanning/tools/CodeQL/status/configurations/actions-FZTWS5DIOVRC653POJVWM3DPO5ZS6Y3PMRSXC3C7MFXGC3DZONUXGLTZNVWA/a24096b7981d3f05871d576ae64dfb71efbcb9d66535f61fa746cea1eb2c8e12